source/docs/articles/sites.md

```diff
@@ -1,33 +1,88 @@
 ---
 title: The Site Dashboard
-description:
+description: Learn how to use the Site Dashboard to build and manage your sites.
 category:
   - developing
-draft: true
-tool_url: https://dashboard.pantheon.io/sites/$PUUID
-tool_image:
+keywords: sites, dashboard, pantheon, get started, settings, new site
 ---
 
-## About-Site
-## Add-Ons
-## Backups
-## Billing-Info
-## Branching Git Workflows for Teams with Multidev
-## Code
+The Site Dashboard is where you can find all the tools you need to successfully build, launch, and manage your site.
+
+![Site Dashboard](/source/docs/assets/images/site-dashboard-image.png)
+
+At the top of the page you'll find all the environments for your site: Dev, Test, and Live. Additional development environments are available with [Multidev](/docs/articles/sites/multidev/).
+
+The **Visit Site** button is available for each environment so you can view the site in each environment. This helpful to view changes in Dev or Test before moving them to Live.
+
+## Team
+The Team tab allows you to change the site owner; add supporting agencies; and add, manage, and delete users. For more information, see [Team Management](/docs/articles/sites/team-management/).
+
+##Settings
+### Plan and Billing
+Here you can select or change your plan and payment method. For more information, see [Selecting a Plan](/docs/articles/sites/settings/selecting-a-plan/).
+
+### Add-Ons
+Pantheon offers several add-ons to help you build dynamic sites. These include [New Relic](docs/articles/sites/newrelic/new-relic-performance-analysis/), [Apache Solr](/docs/articles/sites/apache-solr/), and [Redis](/docs/articles/sites/redis-as-a-caching-backend/).
+
+###Delete Site
+If you no longer need your site, you can remove it here.
+**Note**: This is a permanent change and once you delete it your site cannot be restored.
+
+###About Site
+This tab shows general information about your site, such as the framework, upstream, date it was created, and your current plan.
+
+###PHP Version
+The site default PHP version is applied to every environment unless a PHP version is explicitly selected for each environment. Learn more about [Toggling Between PHP Versions](/docs/articles/sites/settings/toggling-between-php-versions/).
+
+## Support
+If you need assistance, our expert Customer Support team is here to help make building and managing your sites easier. You can contact Support by opening a ticket from your Dashboard. Learn more about our [scope of support](/docs/articles/scope-of-support/).
+
+##Public
```

Comment:
Rename as security indicator (Public/Locked) really a link to the Security tool documented below. 
![image](https://cloud.githubusercontent.com/assets/646061/7438124/fa4c59c0-f013-11e4-8339-cf822d8a52c1.png)